src/core/actors/Tribe.java

```diff
@@ -230,117 +205,89 @@ public Vector2d getPosition()
         return null;
     }
 
-
-    public boolean isMonumentBuildable(Types.BUILDING building)
-    {
-        return monuments.get(building) == MONUMENT_STATUS.AVAILABLE;
+    public ArrayList<Types.TRIBE> getTribesMet(){
+        return tribesMet;
     }
 
-    public void monumentIsBuilt(Types.BUILDING building)
-    {
-        monuments.put(building, MONUMENT_STATUS.BUILT);
-    }
+    public void meetTribe(GameState gs, int tribeID){
 
-    public int getnumKills() {
-        return nKills;
-    }
+        Board b = gs.getBoard();
+        Tribe[] t = gs.getTribes(); // get tribes from boards
 
-    public void addKill() {
-        this.nKills++;
+        boolean[] inMetTribes = new boolean[4];
```

Comment:
We can't use constants like this. We know how many tribes/players we have - so let's use that.